tests/com.espressif.idf.ui.test/src/com/espressif/idf/ui/test/AllUITests.java

```diff
@@ -11,8 +11,8 @@
  * @author Kondal Kolipaka <kondal.kolipaka@espressif.com>
  *
  */
-@RunWith(Suite.class)
-@Suite.SuiteClasses({ EspressifMenuTest.class })
+//@RunWith(Suite.class)
+//@Suite.SuiteClasses({ EspressifMenuTest.class })
```

Comment:
same here, could we remove commented code?

Reply:
Done